src/styles/common/_buttons.scss

```diff
@@ -62,7 +114,8 @@
     }
 
     &.is-loading {
```

Comment:
Do all of the new variants work with the loading and radar treatments?

Reply:
yes they do

Reply:
<img width="162" alt="Screen Shot 2021-11-05 at 3 16 16 PM" src="https://user-images.githubusercontent.com/1571667/140584509-23fbaec8-812c-4db5-aee6-355d6554af5c.png">

with all of the above